frontend/views/containers/dashboard/GroupMembersActivity.vue

```diff
@@ -0,0 +1,160 @@
+<template lang='pug'>
+.c-wrapper
+  .c-columns
+    .c-column
+      i18n.is-title-3(tag='h2') Streaks
+      i18n.has-text-1.c-para Information about your pledges streaks and other streak members appears here.
+
+      ul.spacer
+        li.c-item-wrapper
+          .c-item
+            .icon-star.icon-round.has-background-success.has-text-success
+            .c-item-copy
+              i18n(
+                :args='{ ...LTags("strong"), timeframe: "2 weeks", pourcent: 100 }'
+              ) Group has a streak of {strong_} {pourcent}% Support of {timeframe}{_strong}
+
+        li.c-item-wrapper
+          .c-item
+            .icon-star.icon-round.has-background-success.has-text-success
+            .c-item-copy
+              tooltip(direction='bottom-end')
+                i18n.link(
+                  :args='{  members: 3 }'
+                ) {members} members
+                template(slot='tooltip')
+                  div Rosalía
+                  div Ken M
+                  div Ines de Castro
+
+              //- Todo: discuss if tooltip better than toggle
+              //- i18n.link(
+              //-   tag='button'
+              //-   :aria-label='L("See members")'
+              //-   :args='{  members: 3 }'
+              //-   @click='openModal("todoSeeMembers")'
+              //- ) {members} members
```

Comment:
So on Figma these are displayed as a toggle, and in this PR they are implemented as a tooltip:

![Screen Shot 2022-09-27 at 5 01 18 PM](https://user-images.githubusercontent.com/138706/192658148-827c4411-413b-4664-80ad-0969f4e779c1.jpg)

@pieer and @leihla - can you guys come to an agreement about what this should be?

Also:

- please consider what might look better when the list is 25+ members long
- note that on Figma the designs include more information, such as how long the streak is. Leihla, do you feel this extra info should be there? If so, do any of the other fields also need extra info? Because currently all of these are implemented simply as a list of names in a tooltip.

Reply:
@taoeffect @pieer I don't mind that it shows in a tooltip instead of the expandable, I was following the same pattern we have on the contributions page. 

Regarding the information of duration of the streak, I do think is important to show, but that can be shown on the tooltip as well. 

@pieer any reason you prefer the tooltip?

Reply:
Few reasons:
- common pattern on the web to have more information on link with tooltip.
- quicker to get the info than with click
- less moving element

I can had the streak, yes

I don't think that the tooltip would look great with 25 members though

Reply:
@pieer I think we can keep the tooltip for now. If we have that big of a list we'll need to change this pattern.